.github/workflows/terratest.yml

```diff
@@ -40,8 +43,7 @@ jobs:
         run: |
           make tests
       - name: Release
-        if: github.event_name == 'push'
```

Comment:
This is intentional @jai ?

Reply:
yes, the semantic release workflow doesn't need this filter. It recognizes on its own what is a PR, what is a Push, and it runs as needed.